plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-orders-stats.php

```diff
@@ -207,8 +229,8 @@ public function test_product_attributes_filter() {
 		$response_orders = $response->get_data();
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( 1, $response_orders['totals']['orders_count'] );
+		$this->assertEquals( 11, $response_orders['totals']['orders_count'] );
```

Comment:
This is `11` now, because previously even when using `is_not` it would filter out any orders with simple products. Now the query actually works correctly for `is_not`.